content/courses/advanced-ada/chapters/subprograms_modularity.rst

```diff
@@ -0,0 +1,91 @@
+Subprograms and Modularity
+==========================
+
+.. _Adv_Ada_Private_Subprograms:
+
+Private subprograms
+-------------------
+
+We've seen :ref:`previously <Adv_Ada_Private_Packages>` that we can declare
+private packages. Because packages and subprograms are both library units, we
+can declare private subprograms as well. We do this by using the :ada:`private`
```

Comment:
"are both library units" => "can both be library units"
in other words they are not necessarily library units, even though that's typically the case

Reply:
Thanks, I've just corrected it.